frontend/src/index.css

```
@@ -144,7 +144,107 @@
   }
 }
 
-@layer utilities {
+/* ===== Typography ===== */
+
+@utility typo-3xl-b {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)